### PR TITLE
Annotate flaky tests

### PR DESCRIPTION
**Pull request by igorsyl**
Adds FLAKY_TEST_FIX_ME to flaky tests to improve dev productivity and encourage fixes.

### CODE REVIEW

**Comment by igorsyl** [comment]
Tagged:

```
tests::neon_integrations::mining_events_integration_test
tests::neon_integrations::microblock_large_tx_integration_test
```

Any others?

File: .github/workflows/bitcoin-tests.yml [hunk 1]
```diff
@@ -48,7 +48,7 @@ jobs:
           - tests::neon_integrations::bitcoind_forking_test
           - tests::neon_integrations::should_fix_2771
           - tests::neon_integrations::pox_integration_test
-          - tests::neon_integrations::mining_events_integration_test
+          - tests::neon_integrations::mining_events_integration_test_FLAKY_TEST_FIX_ME
```
Comment (jcnelson) [inline .github/workflows/bitcoin-tests.yml:51]: This is *not* flaky.  It's broken in `develop` due to a recent change in how we report skipped transactions to the event observer.

Comment (kantai) [reply .github/workflows/bitcoin-tests.yml:51]: I'd be happy to approve this with this just marked `mining_events_integrations_test_BROKEN`

Comment (jcnelson) [reply .github/workflows/bitcoin-tests.yml:51]: It's about to be fixed in https://github.com/stacks-network/stacks-blockchain/pull/3204

**Review by kantai — APPROVED** [review]

**Review by jcnelson — CHANGES_REQUESTED** [review]
`mining_events_integration_test` is now fixed in `develop`, so it shouldn't be treated as broken.

**Review by jcnelson — APPROVED** [review]

**Pull request merged**